api/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+server:
+  port: 8080
+  servlet:
+    context-path: /
+
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: http://localhost:8180/auth/realms/SpringBootKeycloak
```

Comment:
Actually I don`t understand how its supposed to work when u specify server.port=8080 and issuer-uri is on 8180
Do u have two separate apps?